GunplaOnlineShop/GunplaOnlineShop/appsettings.Development.example.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
don't delete this configuration file, this is for other people who can use to copy paste and configure their own `appsettings.Development.json` file and run it on their own machine